src/Core/Application/Dependency/TaskLoaderInterface.php

```diff
@@ -5,16 +5,16 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace SprykerSdk\Sdk\Core\Application\Dependency\Repository;
+namespace SprykerSdk\Sdk\Core\Application\Dependency;
```

Comment:
Discussed with @pavelmaksimov25. Don't need expose infrastructure into application layer. Everything should be moved into infrastructure layer from application. Interfaces in dependency folder are not used in application layer (only implemented).

Reply:
Fixed